backend/src/main/java/com/ody/notification/domain/types/AbstractNotification.java

```diff
@@ -35,7 +35,9 @@ private static LocalDateTime calculateSendAt(LocalDateTime sendAt) {
         return sendAt;
     }
 
+    abstract NotificationType getType();
+
     public Notification toNotification() {
-        return new Notification(id, mate, type, sendAt, status, fcmTopic);
+        return new Notification(null, mate, type, sendAt, status, fcmTopic);
```

Comment:
[제안]
Notification id를 받지 않은 생성자에서 null을 넣어주고 있으니 제거해주어도 되겠네요! 괜찮으시면 이 부분도 제가 추후에 반영해두겠습니다~
```suggestion
        return new Notification(mate, type, sendAt, status, fcmTopic);
```